src/Autocomplete/src/OptionsAwareEntityAutocompleterInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Autocomplete;
+
+/**
+ * Interface for classes that will have an "autocomplete" endpoint exposed with a possibility to pass additional options.
+ */
+interface OptionsAwareEntityAutocompleterInterface extends EntityAutocompleterInterface
+{
+    /**
+     * @param array<string, scalar> $options
```

Comment:
In practice, we'll pass an array with scalar values. But as this is passed as the `options` argument when creating a form, there is no technical requirement to pass a scalar, right? I'd remove the php doc entirely. Or remove the `@param` and add a description that these options will be passed when creating the form.